x/logic/interpreter/fs/wasm.go

```diff
@@ -0,0 +1,68 @@
+package fs
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/okp4/okp4d/x/logic/types"
+)
+
+const (
+	queryKey = "query"
+	scheme   = "cosmwasm"
+)
+
+type WasmFS struct {
```

Comment:
It's not a FS as I understand. It's the implemntation of a Hander, right ?

By the way,If the type must obey an interface make an assignment on a blank identifier with a cast on the interface, for type safety.